demisto_sdk/commands/postman_codegen/resources/config-urlscanio.json

```diff
@@ -0,0 +1,822 @@
+
```

Comment:
Is this a test file? If so, let's put it in `test_files` dir

Reply:
It is not, it is a sample file. I am referencing to this file from docs.